lib/trento/infrastructure/commanded/roll_up/roll_up.ex

```diff
@@ -1,12 +1,12 @@
-defmodule Trento.RollUp do
+defmodule Trento.Infrastructure.Commanded.RollUp do
```

Comment:
Following the rules, this should be `Commanded.RollUp.RollUp`, but I understand using it once looks better

Reply:
Yeah it's true, it just feels better but I think we need to adhere to the convention, having this exceptions in the beginning could lead to bad example/direction in the long term, wdyt?

Reply:
So, let's rename it to `RoullUp.RollUp` them